{{cookiecutter.plugin_name}}/.pre-commit-config.yaml

```
@@ -13,7 +13,7 @@ repos:
     rev: 4.0.1
     hooks:
       - id: flake8
-        additional_dependencies: [flake8-typing-imports==1.7.0]
+        additional_dependencies: [flake8-typing-imports==1.12.0]
```

```suggestion
        additional_dependencies: [flake8-typing-imports]
```

since this doesn't get bumped by pre-commit.ci, let's just leave off the version

---

As this coockiecutter template requires python 3.8 or higher, I would prefer version 1.9.0 or higher of `flake8-typing-imports` as versions before that do not support python 3.8.0. Support for python 3.8.0 was added in this commit https://github.com/asottile/flake8-typing-imports/commit/58f1cd69c6963ba6cfd1069e76bd7cac78dee8b2
Is it possible to add something like `flake8-typing-imports>=1.9.0`?

---

sure, that's fine too.